app/controllers/people_controller.rb

```diff
@@ -226,7 +226,7 @@ def destroy
       MarketplaceService::Listing::Command.delete_listings(@current_user.id)
 
       communities.each { |community_id|
-        PaypalService::API::Api.accounts_api.delete(community_id: @current_community.id, person_id: @current_user.id)
```

Comment:
I don't understand why current master doesn't fail tests on Travis because of this...